api/server.py

```diff
@@ -42,6 +45,32 @@
 
 ADMIN_ROLE_NAME = "vz-admin"
 
+CORS_URL = "*"
```

Comment:
We want this wide-open CORS, right? I don't have a good intuition about this. I always end up putting down a wildcard when asked, but it's always to make it work for me, not make it correctly configured.

Reply:
I was tending to agree that wildcard is fine, but like I mention elsewhere, maybe we should tighten up our game and specify our referrer hostnames?